guides/deep-dives/field-editors.md

```diff
@@ -1,7 +1,119 @@
 
-Firstly I am going to create a new field type called `ssh-key` which is essentially the same as a @cardstack/core-types::string so the implementation of the field type is not important for this example. I am creating the field type directly in our Ember App **test-harness** because it is a cardstack plugin (see description of package.json above), so the file name is `deck/test-harness/cardstack/field-types/ssh-key.js`. The contents of this file are exactly the same as the @cardstack/core-types::string implementation.
+Most websites need a way for users to create new content or modify it. For example, a bank customer updates their contact information, a restaurant owner adds more menu items, and a case manager changes the status of a request.
+To handle this, most traditional web apps are littered with forms that are custom-built and individually maintained.
 
-To make use of my new ssh-key field type I need to just define a field with the type: `test-harness::ssh-key` which is constructed of: `<the name in the package.json for test-harness>::<file name of ssk-key>`. If you add this field type to a card type (article for example) it won't show up in the editor until it is used in the `isolated.hbs` file. This is because the field editor toolbox uses the template to identify the parts of the content that are editable (see CS-Field below). Also it is worth noting that there is nothing inherently special about the isolated view of the card when it comes to editing, the only reason why you need to include the field in the isolated templates right now is because we don't have a method to edit embedded cards yet in cardstack.
+Instead, Cardstack developers set some minimal configuration and get instant content editing tools in the browser. For example, the right hand toolbar in the screenshot below is auto-generated from a Card's schema:
 
-When you try to edit the ssh key in the cardstack editing toolbox you will see an error `No such field editor field-editors/ssh-key-editor`. To make a field editor that will be picked up by the system to edit the ssh key all you need to do is create a component `field-editors/ssh-key-editor` that is part of your app in any way i.e. it can be exposed from **any** of the Ember addons or exist in the test-harness directly. As long as there is any component that resolves to that name then it will be used. Also it is worth noting that there is nothing special about this field editor component, the only thing that is specific about its implementation are the parameters that are passed into it as part of the editor toolbox (i.e. its protocol) but I don't know enough about this to detail that API right now.
+![An editor panel on the right edge, with field inputs](/images/card-sdk/right-edge-example.jpg)
+
+It is a WYSIWYG (what you see is what you get) experience where a user makes changes to the Card's content and sees immediately what the result would look like.
+
+## Enabling the Right Edge
```

Comment:
Technically, this really brings in all of the edges, can't enable right edge on its own.

Reply:
The redesigned edges version looks a little different though. Here's the PR: https://github.com/cardstack/cardstack/pull/754. Surprised to see it still green.

<img width="1042" alt="edges-updated" src="https://user-images.githubusercontent.com/16160806/60904871-de708200-a241-11e9-842e-4902756532b6.png">

Reply:
I opened an issue to track updating these screenshots, which follows updating Cardboard